packages/compress-brotli/src/index.js

```diff
@@ -19,12 +19,12 @@ class KeyvBrotli {
 		};
 	}
 
-	compress(value, options) {
-		return this.opts.compress(value, options);
+	compress(value) {
```

Comment:
@jaredwray we need this `options` argument in `compress` or `decompress` methods as `brotli` and `gzip` also have those options filed.

https://github.com/Kikobeats/compress-brotli/blob/master/src/index.js#L29
https://github.com/nodeca/pako/blob/master/lib/inflate.js#L101

Reply:
@alphmth - the goal is that you would pass the options via the constructor for this compression option. 

in the constructor we go with 

```js
	constructor(options) {
		this.opts = options;
	}
```
then we are able to use that for each of the calls. 
```js
	async decompress(value) {
		const {value, expires} = this.compressBrotli.deserialize(data);
		return {value: await this.compressBrotli.decompress(value), expires};
	}
```